apps/jobbrowser/src/jobbrowser/tests.py

```diff
@@ -910,6 +910,11 @@ def app(self, job_id):
       u'app': MockResourceManagerApi.APPS[job_id]
     }
 
+  def kill(self, job_id):
+    job_id = job_id.replace('job', 'application')
```

Comment:
Do we need to move the `job`->`application` replace in the `kill_job()`?

Reply:
Not needed at kill_job(), as the view code checks for instance of yarn and would do the conversion.  
This replace was needed because of local instance of Job running same as the code under line number 1027

1027   def kill(self, job_id):
1028     job_id = job_id.replace('job', 'application')
1029     MockResourceManagerApi.APPS[job_id]['state'] = 'KILLED'
1030     return {}